C/CUDNN/build_tarballs.jl

```diff
@@ -66,11 +64,6 @@ for cuda_version in cuda_versions
     for (platform, sources) in platforms_and_sources
         augmented_platform = Platform(arch(platform), os(platform); cuda=cuda_tag)
         should_build_platform(triplet(augmented_platform)) || continue
-        if platform == Platform("x86_64", "windows")
```

Comment:
Why remove this? https://docs.nvidia.com/deeplearning/cudnn/install-guide/index.html#install-zlib-windows

Reply:
It won't matter _here_ because we don't dlopen the main library, or initialize any of its plugins. So it will never fail, despite being required.

Reply:
Right - I had forgotten why that part was actually there. Rolled back to version which includes the custom zlib.